aten/src/ATen/native/Copy.cpp

```diff
@@ -26,6 +27,53 @@ namespace {
 
 using namespace at;
 
+bool fbgemm_copy_transpose_valid(const Tensor& self, const Tensor& src) {
```

Comment:
The function name could be more meaningful if you include the key conditions , like is_large_size_transpose_copy().   The check has nothing to do with fpgemm_copy.